approved/v93k/flow_control.tf

```diff
@@ -1655,50 +1655,28 @@ test_flow
   }
   else
   {
-    @T1_BEA7F3B_FAILED = 1;
-  }
-  if @T1_BEA7F3B_FAILED == 1 then
-  {
     run_and_branch(erase1_BEA7F3B)
     then
     {
     }
     else
     {
-      stop_bin "", "fail", , bad, noreprobe, red, 12, over_on;
```

Comment:
For this example flow:
  log "Test that if_failed works"
  func :read1, id: :t1, bin: 10
  func :erase1, if_failed: :t1, bin: 12

Does the fact that erase1 uses if_failed: :t1, make read1 a continue on fail?  Or does the bin: 10 have priority on the erase1 failure?